### DISCUSSION TITLE
Support Appstream delivery of the app for software centers

### DISCUSSION THREAD

**Discussion by Odi-3** [mullvad/mullvadvpn-app#9080] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

*I'm probably not in the right repository*

Please add Appstream data for software centers (Linux) and add instructions on the Mullvad website to enable it.

### Alternative solutions

The only way is the one previously indicated.

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [X] Linux

**Comment by faern** [comment]
Please describe what Appstream is and why you want it? This feature request does not outline why it would be an improvement to the current situation.

**Comment by Odi-3** [comment]
> Please describe what Appstream is

https://www.freedesktop.org/wiki/Distributions/AppStream/

> why it would be an improvement to the current situation.

The Mullvad VPN application would appear in the software center with all the information and images. Furthermore it would be possible to install/uninstall simply with one click.
